sources/projects/base64/description.md

```diff
@@ -0,0 +1,22 @@
+This program can encode text to base64-encoded text, and it can decode
+base64-encoded text into normal text.
+
+Base64 is a popular method of encoding strings and other data. It can encode
+images, text, JSON, and almost any other format as well.
+
+It is also often URL-safe.
+
+Base64 works by splitting the data into 6-bit segments, which are then mapped
+to a character in the table.
+
+For example, let's use the string `six`.
+
+This would be split into `1110011 1101001 1111000`. However, these are 8-bit
+groups. Base64 needs 6-bit groups. So this turns into `111001 111010 011111
+000` (in decimal: `57 58 31`).
```

Comment:
```suggestion
This would be split into `01110011 01101001 01111000`. However, these are 8-bit
groups. Base64 needs 6-bit groups. So this turns into `011100 110110 100101 111000` (in decimal: `28 54 37 56`).
```